docs/topics/Disaster-Recovery.md

```diff
@@ -16,53 +16,107 @@ Backups are done with `pg_dump` and restores are done with `psql` ([see here](ht
 
 ### Backup the database
 
-Backup the database into a zip file.
-
-```Shell
-docker exec [container] pg_dump \
--c \
--U postgres \
--d postgres | gzip > /my/dir/dump.gz
-```
+<tabs group="os">
+    <tab id="Windows-backup" title="Windows" group-key="Windows">
+        Backup the database into a zip file.<br/>
+        <code-block>
+        docker exec [container] pg_dump `
+        -c `
+        -U postgres `
+        -d postgres | gzip > /my/dir/dump.gz
+        </code-block> 
+        <tip>
+            gzip needs to be present in the host machine. Alternatively, Windows users can use the Linux/macOS command instructions if executing through the git-bash utility.
+        </tip>
+    </tab>
+    <tab id="Else-backup" title="Linux &amp; macOS" group-key="Else">
+        Backup the database into a compressed archive.<br/>
```

Comment:
Delete line 21 and move this line outside of the `tabs` element. Remove the `<br/>` when you do that too.